provisioning/miniprov/hare_mp/main.py

```diff
@@ -389,10 +401,27 @@ def start_mkfs(hostname: str, hare_config_dir: str):
                           kv_file=f'{hare_config_dir}/consul-kv.json')
     cmd = '/usr/libexec/cortx-motr/motr-mkfs'
     # start mkfs for confd, ios services
+    start = perf_counter()
+    cmd_list = []
     for svc in ('confd', 'ios'):
         svc_fids = generator.get_svc_fids(svc)
         for fid in svc_fids:
-            execute([cmd, fid, '--conf'])
+            # real command line
+            logging.info(f'fid = {str(fid)}\n')
+            cmd_line = [cmd, fid, '--conf']
+            cmd_list += [cmd_line]
+
+    ret_values = []
+    with concurrent.futures.ProcessPoolExecutor() as executor:
+        ret_values += [executor.map(start_mkfs, cmd_list)]
+    finish = perf_counter()
+    perf_result = str(finish - start)
+
+    if (-1 in ret_values):
```

Comment:
-1 seems too specific, why not check `if 0 not in ret__values:`?

Reply:
Hi @mssawant ,
"if 0 not in" is not suitable here as the list can have 0 and -1s.
I have not done this change.
Of course it is possible to do this either by 
1. having an iterative check method in the list (but will not be optimal)
2. OR we can have True/False return values from start_mkfs and simply use it in the if condition (but I did not prefer it as we generally have not done like that I guess)

Reply:
Right, instead of `if 0 not in` we need to use `if not all(rc == 0 for rc in result)`

Reply:
ok. Done!